apps/io-services-cms-webapp/ServiceReviewChecker/function.json

```diff
@@ -1,12 +1,12 @@
 {
   "bindings": [
     {
-      "schedule": "* * * * *",
+      "schedule": "0 9-18 * * 1-5",
```

Comment:
**suggestion(non-blocking):** I would recommend staying broader with the time window.
Is the time local or not?
```suggestion
      "schedule": "0 8-19 * * 1-5",
```